.eslintignore

```diff
@@ -13,6 +13,7 @@ apps/demo-react-cra
 apps/demo-react-vite
 apps/create-fuels-counter-guide
 apps/docs-snippets2/src/typegend
+apps/docs-snippets2/src/**/*.test.ts
```

Comment:
Is the same required for prettier?

Reply:
That was already added in @Torres-ssf 's [PR](https://github.com/FuelLabs/fuels-ts/pull/3238/files#diff-663ade211b3a1552162de21c4031fcd16be99407aae5ceecbb491a2efc43d5d2R30)